### PR TITLE
Setup Leaderboard PR

### PR DESCRIPTION
**Pull request by Aseadd**
### **Commits Done So Far**

✔️ Setup Webpack
✔️ Setup Gitflow
✔️ Add plain HTML file
✔️ Add ES6 modules

### CODE REVIEW

**Comment by Aseadd** [comment]
### **Commits Done So Far**

✔️ Setup Webpack
✔️ Setup Gitflow
✔️ Add plain HTML file
✔️ Add ES6 modules

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+- Webpack
+- API
+
+## Authors
+
+👤 **Addis Tsega**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```
Comment (elmar8287) [inline README.md]: - [ ] OPTIONAL Please, consider adding at also the Setup section - how to set up your project on the local machine (hint `git clone` command)

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Create job done with setup the project 👍 

## Required Changes ♻️

- [ ] You PR ready for APPROVE, but there is only one issue you have to fix:
- Please, consider fixing the PR description. You added it in a comment, but you have to put it in the PR description section. Here is some instruction on how to do it:
Screen Shot 2022-08-24 at 12 46 04


## Optional suggestions

- Please, find my OPTIONAL suggestion under the review 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Aseadd,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**